### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by BlueDragn** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#37] [Outreachy 2022] [answered]
Hello everyone, I am Anshuman from India, a web developer. I would love to contribute to this project.

**Comment by BatoolMM** [comment, marked as answer]
Welcome @BlueDragn to the community!

We have two projects for Outreachy, you can find them here:
1- [Improve the bilinguality and accessibility of the website](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/issues/12)
2- Improve Localisation for [Open Educational Resources](https://github.com/Open-Science-Community-Saudi-Arabia/localisation/issues/13).

Feel free to comment under the issue, if you have any questions!
